src/app/components/toast/index.html

```diff
@@ -0,0 +1,92 @@
+    <sky-demo-page-property
+      propertyName="openMesage(message, config?)">
+      Opens a message toast and displays the provided message. The <stache-code>message</stache-code> parameter is of type <stache-code>string</stache-code> and determines the message to be displayed. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="openTemplatedMessage(customComponentType, config?, providers?)">
+      Opens a custom toast and renders the provided component in a toast. The <stache-code>customComponentType</stache-code> parameter is of type <stache-code>Type&lt;SkyToastCustomComponent&gt;</SkyToastCustomComponent></stache-code> and determines the component to be rendered. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. The optional <stache-code>providers</stache-code> parameter is of type <stache-code>Provider[]</stache-code> and specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastConfig type">
+    <sky-demo-page-property
+      propertyName="message"
+      isOptional="true">
+      Sets the message that the toast displays. This may not be provided with the <stache-code>customComponentType</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="customComponentType"
+      isOptional="true">
+      Sets the component that the toast renders. This may not be provided with the <stache-code>message</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="providers"
+      isOptional="true">
+      Specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="toastType"
+      isOptional="true">
+      Sets the type of the toast and determines the color and icon displayed. Available values are: <em>info, success, warning, and danger</em>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastMessage type">
+     <sky-demo-page-property
+      propertyName="close()">
+      Closes the linked toast and activates its <stache-code>closing</stache-code> and then <stache-code>closed</stache-code> states.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isClosing">
+      An observable boolean showing if the toast is currently closing.
```

Comment:
I'd change "showing if" to "that indicates whether." (Just a trivial wording preference for consistency, so no big deal if you don't want to change this. ... I'd also delete "currently."